core/js/src/main/scala/java/text/Normalizer.scala

```diff
@@ -0,0 +1,25 @@
+package java.text
+
+import scala.scalajs.js
+
+object Normalizer {
+
+  @inline def normalize(src: CharSequence, form: Form): String =
```

Comment:
should this be `@inline`? we should also do a double check on throwing the NPE

Reply:
The NPE I got from the Javadocs: https://docs.oracle.com/javase/8/docs/api/java/text/Normalizer.html#normalize-java.lang.CharSequence-java.text.Normalizer.Form-

Regarding `@inline`, yes, this enables the `Normalizer` object to be completely elided in the generated JS. It's a common pattern in the Scala.js java lib implementation.

Reply:
I was thinking about https://github.com/scala-js/scala-js/pull/4167#discussion_r478887241 - I need to get sjrd to clarify that for me sometime :)

Reply:
He's referring to this:
> Because Scala.js does not receive VM support to detect such erroneous conditions, checking them is typically too expensive.
https://www.scala-js.org/doc/semantics.html

AFAICT the Javalib in Scala.js does sometimes explicitly check for nulls, but not everywhere since that would be far too expensive. I'm not sure what the right thing to do is here, what do you think?

Reply:
I'm fine with this, thanks a lot! and thanks for the additional education.